.circleci/config.yml

```diff
@@ -161,7 +161,6 @@ jobs:
           aws_secret_access_key: $AWS_SECRET_ACCESS_KEY
     working_directory: /app
     steps:
-      - run: node scripts/strictness-migration.js check-pr
```

Comment:
[L170](https://github.com/artsy/force/pull/12696/files#diff-78a8a19706dbd2a4425dd72bdab0502ed7a2cef16365ab7030a5a0588927bf47L170) runs the same script so removed this redundancy.

Reply:
This can actually be entirely removed! We've already migrated the whole codebase.

Reply:
Do you mean the whole `check-pr` job or the other `node scripts/strictness-migration.js check-pr` run ?

Reply:
Rereading, I assume you mean the `run node scripts/strictness...`
If so, should I remove the [script](https://github.com/artsy/force/blob/main/scripts/strictness-migration.js) too?

Reply:
yup and yup

Reply:
Done in https://github.com/artsy/force/pull/12696/commits/b00850551427f3e0b1271c323654b3b3f7818343